src/main/java/com/synopsys/integration/alert/web/model/FieldModel.java

```diff
@@ -35,6 +36,16 @@
     private final String descriptorName;
     private final String context;
 
+    public FieldModel() {
+        this("", "", new HashMap<>());
```

Comment:
What is this case for? I'm a little wary of initializing a String that represents an enum to anything other than a valid value or null.

Reply:
Blank constructor is for Gson serialization. I made these private and inserted null instead of values.